features/kubernetes-nmstate/README.md

```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
 For testing you can use the `./testing.sh` script in this folder.
 The test will create a bond interface with the name `bond1` and attach `ens8` and `ens9` as the slave interfaces.
 
-*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
\ No newline at end of file
+*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
+
+### Node configuration examples
+
+```
+# IP configuration
+
+10: bond1: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
```

Comment:
IDK if you want to remove the MAC addresses here

Reply:
masked both macs and IPs.
fyi, there are mac and IP addresses listed in NodeNetworkState section also, please open an issue so the right person can address that too.